src/View/mainpage/DrawROIWindow.py

```diff
@@ -1,22 +1,21 @@
 import csv
 import math
 
-from PyQt5 import QtCore, QtGui, QtWidgets
-from PyQt5.QtGui import QIcon, QPixmap
-from PyQt5.QtCore import QCoreApplication, QThreadPool
-from PyQt5.QtWidgets import QWidget, QTreeWidget, QTreeWidgetItem, QMessageBox, QHBoxLayout, QVBoxLayout, \
-    QLabel, QLineEdit, QSizePolicy, QPushButton, QDialog, QListWidget, QGraphicsPixmapItem, QGraphicsEllipseItem
+from PyQt5.QtGui import QIcon
+from PyQt5.QtWidgets import QMessageBox, QHBoxLayout, QLineEdit, QSizePolicy, QPushButton, QDialog, QListWidget, QGraphicsPixmapItem, QGraphicsEllipseItem
 from PyQt5.Qt import Qt
-import os
-from src.Model import ROI
-from src.View.mainpage.DicomView import *
-import matplotlib.pyplot as plt
 
+from src.Controller.MainPageController import MainPageCallClass
+from src.Model.PatientDictContainer import PatientDictContainer
+from legacy.src.View.mainpage.DicomView import *
```

Comment:
are we relying on legacy?
If not, get rid of this import, or else it makes it seem like we have to keep legacy, and odds are we don't want to.